remote/src/components/NavBar.jsx

```diff
@@ -3,9 +3,9 @@ import { Link, Outlet } from "react-router-dom";
 function NavBar() {
```

Comment:
The name of this component is misleading. can we please change it?

Reply:
Done